terraform/amazon/templates/instance_pools/instance_pool_instance.tf.template

```diff
@@ -236,4 +236,16 @@ resource "aws_route53_record" "{{.TFName}}" {
   count   = "${var.{{.TFName}}_min_count}"
 }
 
+{{ if eq .Role.Name "etcd" -}}
+resource "aws_route53_record" "{{.TFName}}-srv-record" {
+  zone_id = "${var.private_zone_id}"
+  name    = "{{.Role.Name}}-svc.${data.template_file.stack_name.rendered}"
```

Comment:
I think maybe we call it `-exporters` instead of `-svc`

Reply:
Should be fixed in d356961bd36590a716fc8da9f1bb1124d6387739